runners/samza/job-server/build.gradle

```diff
@@ -32,7 +32,6 @@ applyJavaNature(
 )
 
 dependencies {
-  compile project(":runners:samza")
```

Comment:
If this is the same as the Flink and Spark job servers, this dependency is actually needed somehow (I haven't figured out how yet). So I think it's fine to keep the compile dependency and add `permitUnusedDeclared` for now.

Reply:
RIght, this is the same as Flink and Spark job server. I was using runShadow task to bring up the job server locally, and plan to add portability tests later.